ecs-logs-collector.sh

```diff
@@ -246,6 +247,8 @@ get_mounts_info()
     vgs > ${info_system}/vgs.txt
   fi
 
+  vmstat 1 5 > ${info_system}/vmstat.txt
```

Comment:
This is a great additional data point to gather, thanks! I think this ought to be broken out into its own function complete with a note saying its going to look idle for a moment.